### PR TITLE
PR template adjustment, undefault validation_short

### PR DESCRIPTION
**Pull request by pfuehrlich-pik**
## :bird: Description of this PR :bird:

- validation_short.R is no longer run per default after each run
- new start/extra script "compile_main_gms.R" which runs download_data and `gams main.gms action=c` (running only the latter fails in freshly cloned repos due to missing input data)
- PR template: checkbox for RSE/content review, clarification
- the new PR template looks like [this](https://github.com/pfuehrlich-pik/magpiemodel/blob/PR-template/.github/pull_request_template.md)

## :wrench: Checklist for PR creator :wrench:

- [x] Label pull request [from the label list](https://github.com/magpiemodel/magpie/labels).
  - **Low risk**: Simple bugfixes (missing files, updated documentation, typos) or changes  in start or output scripts
  - **Medium risk**: Uncritical changes in the model core (e.g. moderate modifications in non-default realizations)
  - **High risk**: Critical changes in model core or default settings (e.g. changing a model default or adjusting a core mechanic in the model)

- [x] Self-review own code
  - No hard coded numbers and cluster/country/region names.
  - The new code doesn't contain declared but unused parameters or variables.
  - [`magpie4`](https://github.com/pik-piam/magpie4) R library has been updated accordingly and backwards compatible where necessary.
  - `scenario_config.csv` has been updated accordingly (important if `default.cfg` has been updated)

- [x] Document changes 
  - Add changes to `CHANGELOG.md`
  - Where relevant, put In-code documentation comments
  - Properly address updates in interfaces in the module documentations
  - run [`goxygen::goxygen()`](https://github.com/pik-piam/goxygen) and verify the modified code is properly documented

- [x] Perform test runs
  - **Low risk**: 
    - Run a compilation check via `gams main.gms action=c`
  - **Medium risk**: 
    - Run test runs via `Rscript start.R --> "test runs"`
  - **High risk**:
    - Run test runs via `Rscript start.R --> "test runs"`
    - Default run based on the current version of the fork from which PR is made
    - Provide relevant comparison plots (land-use, emissions, food prices, land-use intensity,...)

### :chart_with_downwards_trend: Performance changes :chart_with_upwards_trend:
  
  - Current develop branch default : ** mins
  - This PR's default :  ** mins

## :rotating_light: Checklist for reviewer :rotating_light:

- PR is labeled correctly
- Code changes look reasonable
  - No hard coded numbers and cluster/country/region names.
  - No unnecessary increase in module interfaces
  - model behavior/performance is satisfactory.
- Changes are properly documented
  - `CHANGELOG` is updated correctly
  - Updates in interfaces have been properly addressed in the module documentations
  - In-code documentation looks appropriate

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -23,12 +23,14 @@
 
 - [ ] Perform test runs
   - **Low risk**: 
-    - Run a compilation check via `gams main.gms action=c`
+    - Run a compilation check via `Rscript scripts/start/extra/compile_main_gms.R`
```
Comment (tscheypidi) [inline .github/pull_request_template.md:26]: why should that be started directly and not via `Rscript start.R` This looks inconsistent to me

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -47,3 +49,5 @@
   - `CHANGELOG` is updated correctly
   - Updates in interfaces have been properly addressed in the module documentations
   - In-code documentation looks appropriate
+- [ ] content review
```
Comment (tscheypidi) [inline .github/pull_request_template.md:52]:
```suggestion
- [ ] content review done (at least 1)
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -47,3 +49,5 @@
...
+- [ ] RSE review
```
Comment (tscheypidi) [inline .github/pull_request_template.md:53]:
```suggestion
- [ ] RSE review done (at least 1)
```

File: scripts/start/extra/compile_main_gms.R [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+# |  of MAgPIE and licensed under AGPL-3.0-or-later. Under Section 7 of
+# |  AGPL-3.0, you are granted additional permissions described in the
+# |  MAgPIE License Exception, version 1.0 (see LICENSE file).
+# |  Contact: magpie@pik-potsdam.de
+
+# ------------------------------------------------
+# description: download input and compile main.gms
+# ------------------------------------------------
+
+source("scripts/start/download_data.R")
```
Comment (tscheypidi) [inline scripts/start/extra/compile_main_gms.R:12]: Better not build cross dependency to other start scripts and copy the code of it here instead. (this script should still work even if the other one is being deleted)

Comment (tscheypidi) [reply scripts/start/extra/compile_main_gms.R:12]: In addition, put this script in the start folder, not in the extra-subfolder as this script will be a default one and regularly used. The extra folder is more meant for scripts with more exotic use cases

**Review by tscheypidi — CHANGES_REQUESTED** [review]

**Comment by pfuehrlich-pik** [comment]
The compile check seems to do what it is supposed to. It's just downloading the data and then running `gams main.gms action=c`. Previously the PR template would ask people to run only the `gams` command, but that fails unless the data was downloaded previously.

**Review by tscheypidi — APPROVED** [review]
Looks good

**Pull request merged**